ospfd/ospf_packet.c

```diff
@@ -4319,21 +4320,15 @@ void ospf_ls_ack_send_delayed(struct ospf_interface *oi)
  * punt-to-CPU set on them. This may overload the CPU control path that
  * can be avoided if the MAC was known apriori.
  */
-#define OSPF_PING_NBR_STR_MAX  (BUFSIZ)
 void ospf_proactively_arp(struct ospf_neighbor *nbr)
```

Comment:
I like this, but how is this going to work for *BSDs? Maybe keep it `#ifdef` in that case? Or should it not work there until someone implements the equivalent for route sockets?

I might be missing something, but I'd just like to clarify that.

Reply:
what did you think of my suggestion to just make it call out to ping in the bsd dataplane code until something equivalent exists in bsd? basically move the code that was in ospf to call ping to if_socket.c or whatever.

Reply:
@sworleys yes, I think that is one good solution, otherwise next daemon that requires some form of this will have to re-implement it.

Sorry I missed your comment before.